app/src/org/commcare/android/database/global/models/ApplicationRecordV1.java

```diff
@@ -0,0 +1,38 @@
+package org.commcare.android.database.global.models;
+
+import org.commcare.android.storage.framework.MetaField;
+import org.commcare.android.storage.framework.Persisted;
+import org.commcare.android.storage.framework.Persisting;
+import org.commcare.android.storage.framework.Table;
+
+/**
+ * The previous version of ApplicationRecord in the database,
```

Comment:
Would be good to specify something more concrete than "previous" here, to make it clear that this record will read and make available the model in the V1 database.